src/components/profile/ProfileService.js

```diff
@@ -165,7 +165,7 @@ goog.require('ga_urlutils_service');
           $http(params).success(function(data, status) {
             // When all the geometry is outside switzerland
             if (data.length == 0) {
-              data = [{alts: {COMB: 0}, dist: 0}];
+              data = [{alts: {}, dist: 0}];
```

Comment:
I think we need to keep the structure here, otherwise d3 will get undefines when creating charts. But we should use the
available `elevationModel` variable. Adding the followin on the next line might do the trick:

`data[0].alts['elevationModel'] = 0`